interface/themes/style_light.css

```diff
@@ -40,7 +40,7 @@ a:hover {
     z-index:100;
      }
 
-    .container{
+    .menuBar .container{
```

Comment:
This is a fix for a global issue - we are using `container` in this script which had an extraneous padding/margin on the top of it, but it should have only been applied to containers inside of the menubar